server.js

```diff
@@ -15,40 +24,88 @@ app.use("/public", express.static(__dirname + "/public"));
 app.use(cookieParser());
 
 // get requests
-app.get("/", (req, res) => {
+app.get("/", async (req, res) => {
     const { cookies } = req
-    res.render('index', sessions[cookies.session_id]);
+    try {
+        const currentSession = await Session.findById(cookies.session_id)
+        if (currentSession === null) {
+            res.render('index');
+        } else {
+            res.render('index', currentSession);
+        }
```

Comment:
What's the difference between passing `currentSession` and not passing it? Aren't you able to do just
```suggestion
        res.render('index', currentSession);
```
And if it's null it's null?